docker-compose.yml

```diff
@@ -1,21 +1,24 @@
 version: "3.4"
 services:
-  lodestar:
+  beacon_node:
     build: .
+    image: lodestar
     restart: always
     volumes:
-      - lodestar:/root/.local/share/lodestar
-    command: beacon
+      - beacon_node:/data
+    env_file: .env
```

Comment:
is the `cp default.env .env` step optional?

Reply:
No, it's mandatory. Otherwise docker-compose errors